gregBorchers/Homework2GregBorchers-cards-template1a.js

```diff
@@ -0,0 +1,169 @@
+// Greg Borchers homework 2
+// Simple version (no error-detection)
+
+var deckOfCards =  // Aces[Heart,Diamond,Spade,Club], Twos[Heart,Diamond,Spade,Club]... Eights[....]
+                [ "Ace of Hearts"
+                , "Ace of Diamonds"
+                , "Ace of Spades"
+                , "Ace of Clubs"
+
+                , "Two of Hearts"
+                , "Two of Diamonds"
+                , "Two of Spades"
+                , "Two of Clubs"
+
+                , "Three of Hearts"
+                , "Three of Diamonds"
+                , "Three of Spades"
+                , "Three of Clubs"
+
+                , "Four of Hearts"
+                , "Four of Diamonds"
+                , "Four of Spades"
+                , "Four of Clubs"
+                                
+                , "Five of Hearts"
+                , "Five of Diamonds"
+                , "Five of Spades"
+                , "Five of Clubs"
+
+                , "Six of Hearts"
+                , "Six of Diamonds"
+                , "Six of Spades"
+                , "Six of Clubs"
+
+                , "Seven of Hearts"
+                , "Seven of Diamonds"
+                , "Seven of Spades"
+                , "Seven of Clubs"
+
+                , "Eight of Hearts"
+                , "Eight of Diamonds"
+                , "Eight of Spades"
+                , "Eight of Clubs"
+
+                , "Nine of Hearts"
+                , "Nine of Diamonds"
+                , "Nine of Spades"
+                , "Nine of Clubs"
+
+                , "Ten of Hearts"
+                , "Ten of Diamonds"
+                , "Ten of Spades"
+                , "Ten of Clubs"
+
+                , "Jack of Hearts"
+                , "Jack of Diamonds"
+                , "Jack of Spades"
+                , "Jack of Clubs"
+
+                , "Queen of Hearts"
+                , "Queen of Diamonds"
+                , "Queen of Spades"
+                , "Queen of Clubs"
+
+                , "King of Hearts"
+                , "King of Diamonds"
+                , "King of Spades"
+                , "King of Clubs" ];
+
+// **** alternative to build deckOfCards[] **** 
+// Loop across two string arrays to fill the arrary
+// Did not do this because having the static array was a useful visual reference
+// Here is code that would fill the array
+
+// var suitNames = ["Hearts", "Diamonds", "Spades", "Clubs"];
+// var rankNames = ["Ace","Two","Three","Four","Five","Six","Seven","Eight","Nine","Ten","Jack","Queen","King"];
+// var deckOfCards2 = [];
+// var cardID =0;
+
+// for (var i = 0; i < rankNames.length; i++) {
+//   for (var j = 0; j < suitNames.length; j++) {
+//     deckOfCards2[cardID] =  rankNames[i] + " of " + suitNames[j]
+//     console.log( deckOfCards2[cardID] );
+//     cardID++;
+//   };
+// };
```

Comment:
5) See my comment in problem 3 about keeping your code dry: it would be much less repetition to not have that big array of cards. Having code that autogenerates the array is a good midway point, but the ideal would be to have the card name only exist when it is asked for.